src/main/scala/org/broadinstitute/dsde/workbench/sam/openam/LdapAccessPolicyDAO.scala

```diff
@@ -27,10 +27,13 @@ class LdapAccessPolicyDAO(protected val ldapConnectionPool: LDAPConnectionPool,
   }
 
   override def createResource(resource: Resource): Future[Resource] = Future {
-    ldapConnectionPool.add(resourceDn(resource),
+    val attributes = Seq(
```

Comment:
can you use `List` here, see discussion https://broadinstitute.slack.com/archives/C9ZSW4R7U/p1536949814000100